includes/admin/views/html-admin-page-addons.php

```diff
@@ -62,7 +62,13 @@ class="<?php echo $current_section === $section->slug ? 'current' : ''; ?>"
 
 	<div class="wrap">
 		<div class="marketplace-content-wrapper">
-			<?php if ( ! empty( $search ) ) : ?>
+			<?php if ( count( $addons ) == 0 ) : ?>
+				<h1 class="search-form-title">
+					<?php // translators: search keyword. ?>
```

Comment:
You can remove this since there is no placeholder in the string below.

Reply:
should I make the below change ?
```suggestion
					<?php // translators: search keyword. ?>
```

Reply:
What exactly should I remove

Reply:
He is talking about this `<?php // translators: search keyword. ?>`

Reply:
have made the changes